medical/models/medical_patient.py

```diff
@@ -95,6 +95,15 @@ def action_invalidate(self):
             ('z', 'law marriage'),
         ])
 
+    @api.multi
+    def onchange_state(self, state_id):
+        '''
```

Comment:
Use double quote for docstrings, and start the first doc line in the same line as `"""`